### ISSUE TITLE
Dord [DorD_Sept]

### ISSUE THREAD

**Issue by fehpym** [Rinnegatamante/YoYo-Loader-Vita-Compatibility#451]
# Game summary
- Game name: [Dord](https://gamejolt.com/games/dord/306209)
- Game version: 1.1.3

# YoYo Loader summary
- Version: v 0.1 (0de3e4b)

# Issues
couldn't find a bytecode 16 donor that would work
converting to 17 works with Gun Done, but this seems to break interactions, making it not possible to progress early on

# Screenshots

![2022-05-30-193102](https://user-images.githubusercontent.com/103788734/171285441-698e336e-9a95-4857-bed0-1983182da9b9.png)

# Recommended labels
Ingame -

**Comment by fehpym** [comment]
Yeah, I'm saying none of the byte 16 games on that list worked for me

**Comment by fehpym** [comment]
Update;

Swapped with #35 and disabled audio
unmodified byte 16 droid, but interactions still broken - text boxes trigger repeatedly over and over again and make you stuck in their trigger zone until you slowly wiggle out

**Comment by panpeus** [comment]
Asset swap with Stolen Memory

39e783 | 5aac59 

- In Game High (though fully playable).
- 59fps
- Require keys.ini
- - WASD action keys for in game battle is auto detected and is mapped to the d-pad.
![2023-01-06-055053-663740](https://user-images.githubusercontent.com/103394740/211113786-1249dc59-b915-4f15-ad08-961abeb07566.jpg)

LANALOG=ON

LEFT=LEFT
RIGHT=RIGHT
UP=UP
DOWN=DOWN
CROSS= 
sSQUARE=SHIFT
SELECT=ESC

- Though tedious you can progress in game by mashing the "interact" key (mapped to CROSS 
  on the Vita) until Midy finally floats free of the hit box that triggers dialogs.

# ISSUE

- Text boxes trigger repeatedly in a loop
- Certain animations (rain as an example) is not animated but a still frame.
- Can't trigger event to open "gate" to the next sequence early in the the game. (see below)
- In the beginning of the game, you'll come across three knight armours whom you have to 
  spook to cause them to drop each of their staff, however this does not happen. Once they are 
  spooked, their hands open but the staff does not fall to the floor, which is required 
  in order to trigger an event which will open the "gate" which leads to the next scene. 
![2023-01-05-235031-815016](https://user-images.githubusercontent.com/103394740/211114111-27dd2738-1f05-4883-85bb-dc1a9a3ce572.jpg)

- Found work around to this problem. (see solution).
-  - First Boss
![2023-01-06-161227-484727](https://user-images.githubusercontent.com/103394740/211113994-84e10c87-e0c4-4e0e-945f-c7aba0df8f6c.jpg)

# UTMT

- Run Technical Script and change byte code 16 to 17

# SOLUTION

- Solution; ran game on PC and played past the three knight armours, triggered events and proceeded past "gate"
  until the first save point was reached. Took save data from PC and transferred to the Vita.
-
- Take savedata from inside SAVE_DATA_DorD_Sept.zip and transfer it to ux0:data/gms/Dord/assets
- Launch and "continue" the game. This will skip the story of why Midy is on his quest.
- For those whom, wish to see the story before reaching the first save post; you can watch this [walk-through](https://www.youtube.com/watch?v=W-2dn6HLsYw).
[SAVE_DATA_DorD_Sept.zip](https://github.com/Rinnegatamante/YoYo-Loader-Vita-Compatibility/files/10363695/SAVE_DATA_DorD_Sept.zip)

[keys_DorD_Sept.zip](https://github.com/Rinnegatamante/YoYo-Loader-Vita-Compatibility/files/10363702/keys_DorD_Sept.zip)


[yyl.log](https://github.com/Rinnegatamante/YoYo-Loader-Vita-Compatibility/files/10363698/yyl.log)
